dev-perl/Text-Template/Text-Template-1.530.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DIST_AUTHOR=MSCHOUT
+DIST_VERSION=1.53
+
+inherit perl-module
+
+DESCRIPTION="Expand template text with embedded Perl"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~x86 ~ppc-macos"
+IUSE="test"
+
+RDEPEND="virtual/perl-Carp
+	virtual/perl-Exporter"
+
+DEPEND="${RDEPEND}
+	virtual/perl-ExtUtils-MakeMaker
+	test? (
+		virtual/perl-File-Temp
+		virtual/perl-Safe
+		virtual/perl-Test-Simple
+		dev-perl/Test-More-UTF8
```

Comment:
This list should be sorted similar to how they'd sort if it was just a list of module names. That is, Test-More-UTF8 comes before Test-Simple